circuit_maintenance_parser/output.py

```diff
@@ -132,9 +132,10 @@ class Maintenance(BaseModel):
     status: Status
     start: StrictInt
     end: StrictInt
-    stamp: StrictInt
     organizer: StrictStr
 
+    # Non mandatory attributes
+    stamp: Union[StrictInt, None] = None
```

Comment:
As in #18, suggest using `Optional` here rather than `Union[..., None]`

Reply:
sure, It's fixed after the merge from that PR (I only cherry-picked my original commit)